docs/accessibility/index.md

```diff
@@ -0,0 +1,60 @@
+---
+id: index
+title: Backstage Accessibility
+description: Documentation on Backstage Accessibility
+---
+
+In an effort to bake accessibility practices further into our process of building features for Backstage, we’ve rolled out automated CI tests to the OSS project on some of our core features, Software Catalog, Software Templates, Search and TechDocs. As these are just a few plugins of many out there, we want to encourage you to consider the accessibility implications of your work on Backstage in order to build a great experience for everyone.
+
+## How to contribute
+
+There are multiple ways to contribute to making Backstage accessible to everyone, below we list examples of some ways you can do that.
+
+### Run Lighthouse in CI on your plugin
+
+If your plugin lives in the [Backstage main repository](https://github.com/backstage/backstage/) you can modify the [urls in the Lighthouse config](https://github.com/backstage/backstage/blob/master/lighthouserc.js#L19-L34) to run the Lighthouse checks on urls where your plugin exists as well. E.g.
```

Comment:
To make sure we're referencing lines which aren't subject to drift up and down the file, suggest to point to a specific git  hash rather than a branch. There was recently a PR to refactor that across the docs so it'll help to keep consistent!

Reply:
good call out, as we just discussed that in the thread above and I wanted to at least keep the line specific links as permalinks! Will update!

Reply:
updated in https://github.com/backstage/backstage/pull/17698/commits/1d2932d97194db66a663e5a970c78fa8330718f5